dotcom-rendering/src/web/components/YoutubeBlockComponent.importable.tsx

```diff
@@ -229,7 +232,7 @@ export const YoutubeBlockComponent = ({
 				eventEmitters={[ophanTracking, gaTracking]}
 				pillar={format.theme}
 				origin={process.env.NODE_ENV === 'development' ? '' : origin}
-				shouldStick={stickyVideos}
+				shouldStick={!!stickyVideos}
```

Comment:
I _think_ this is a boolean anyway, it's just that the type definition on line 35 is optional when it doesn't need to be.

Reply:
Good shout thanks. Updated